
#manipulator geometry parameters:
delta_2:
    base_radius:               0.15
    platform_radius:           0.07
    base_joint_spacing:        0.04
    platform_joint_spacing:    0.07
    proximal_link_length:      0.2
    distal_link_length:        0.4